data/human/deep missions.txt

```diff
@@ -2809,7 +2809,7 @@ mission "Home for Skadenga 1"
 			choice
 				`	(Board the Ice-Crawler.)`
 			`	The Ice-Crawler, 'Brunhilda,' smells of oil, atmosphere, and perspiration. The confines are tight, with a padded bench area and threadbare seat-belts. There are wires and hard angles everywhere. You're careful not to get tangled in the green, white and blue wires running overhead and into the cockpit. You take a seat and strap in.`
-			`	It is only a little warmer here than it was in the maintenance dock. To your left and right are wide, scratched, and frost-rimmed windows.`
+			`	It is only a little warmer here than it was in the maintenance dock. To your left and right are wide, scratched, and frost-rimed windows.`
```

Comment:
The original is correct, though awkward.

"Frost-rimed" is not correct in its own right, either.